flawfinder.py

```diff
@@ -54,6 +54,7 @@
 import csv  # To support generating CSV format
 import hashlib
 # import formatter
+from sariflogger import SarifLogger
```

Comment:
Flawfinder doesn't use separate files, so that installation can be as simple as "copy this single file".